src/Form/TricksFormType.php

```diff
@@ -79,6 +106,15 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ]);
     }
 
+    /**
+     * Summary of function configureOptions
+     *
+     * Send an email to the user with a token to verify his account
+     *
+     * @param OptionsResolver $resolver OptionsResolver
+     *
+     * @return void
+     */
```

Comment:
https://symfony.com/doc/current/forms.html#usage

> This class is a "plain-old-PHP-object" because, so far, it has nothing to do with Symfony or any other library. It's a normal PHP object that directly solves a problem inside your application (i.e. the need to represent a task in your application).

Donc normalement tu peux utiliser des Models (se sont classes PHP simple qui représente un concept métier dans ton application et non une classe "technique" de base de données)

Reply:
ok merci je vais regarder ça de plus près !